src/blocks/FooterBlockBrandPivot.tsx

```diff
@@ -0,0 +1,246 @@
+
+const FooterWrapper = styled('div')({
+  padding: `5rem 0`,
+
+  [MOBILE_BP_UP]: {
+    padding: `4.25rem 0 12rem`,
+  },
+})
+
+const IconWrapper = styled('div')({
+  display: 'flex',
+  justifyContent: 'flex-end',
+  margin: '0 auto 3.125rem',
+  paddingLeft: CONTENT_GUTTER_MOBILE,
+  paddingRight: CONTENT_GUTTER_MOBILE,
+
+  [MOBILE_BP_UP]: {
+    paddingLeft: CONTENT_GUTTER,
+    paddingRight: CONTENT_GUTTER,
+  },
+
+  ...SITE_MAX_WIDTH,
+})
+
+const FooterInnerWrapper = styled('footer')({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'center',
+  [TABLET_BP_DOWN]: {
+    flexDirection: 'column',
+  },
+})
+
+const LinkTextWrapper = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+})
+
+const LinksColumnsWrapper = styled('nav')({
+  display: 'flex',
+  flexDirection: 'row',
+  marginTop: '2rem',
+  [TABLET_BP_UP]: {
+    marginTop: '0',
+    marginBottom: '5rem',
+    justifyContent: 'center',
+  },
+  [BP_DOWN]: {
+    flexWrap: 'wrap',
+  },
+})
+
+const ColumnHeader = styled('div')({
+  paddingBottom: '1.5rem',
+  fontSize: '1.125rem',
+  color: colorsV3.gray500,
+  [TABLET_BP_UP]: {
+    paddingBottom: '2rem',
+    fontSize: '1.5rem',
+  },
+})
+
+const LinkColumn = styled('div')({
+  '&:not(:last-of-type)': {
+    paddingRight: '7rem',
+    paddingBottom: '.5rem',
+    '@media(max-width:1100px)': {
+      paddingRight: '4rem',
+    },
+    [BP_DOWN]: {
+      width: '50%',
+      paddingRight: 0,
+      paddingBottom: '2rem',
+    },
+  },
+  '&:last-of-type': {
+    [BP_DOWN]: {
+      paddingBottom: '2rem',
+    },
+  },
+})
+const Link = styled('a')({
+  display: 'block',
+  color: 'inherit',
+  fontSize: '0.875rem',
+  textDecoration: 'none',
+  marginBottom: '0.8625rem',
+})
+
+const FooterFooter = styled('div')({
+  opacity: 0.28,
+  fontSize: '0.9rem',
+  marginTop: '2rem',
+})
+
+type FooterBlockProps = BaseBlockProps
+
+export const FooterBlockBrandPivot: React.FunctionComponent<FooterBlockProps> = ({
+  color,
+  extra_styling,
+}) => (
+  <GlobalStoryContainer>
+    {({ globalStory }) => (
+      <ContextContainer>
+        {(context) => (
+          <SectionWrapper
+            brandPivot
+            colorComponent={color}
+            size="none"
+            extraStyling={extra_styling}
+          >
+            <FooterWrapper>
+              <IconWrapper>
+                <HedvigH size={48} />
+              </IconWrapper>
+              <ContentWrapper>
+                <FooterInnerWrapper>
+                  <LinkTextWrapper>
+                    {globalStory.content.get_started &&
+                    globalStory.content.get_started.length &&
+                    globalStory.content.company &&
+                    globalStory.content.company.length &&
+                    globalStory.content.legal &&
+                    globalStory.content.legal.length &&
+                    globalStory.content.social &&
+                    globalStory.content.social.length ? (
```

Comment:
maybe we could make this slightly prettier with optional chaining?

something like

```js
globalStory.content.get_started?.length
&& globalStory.content.company?.length 
```

Reply:
Indeed!